index.html

```diff
@@ -232,16 +234,29 @@
     email = email.toLowerCase();
     popupContainer.innerHTML = '<div class="teacher"><div>' + '<img src="' + imgSrc +
         '" align="right" class="fullsizeimage"></img><p>' + teacherObj.bio + '</p></div>' +
-        '<schedule-lite></schedule-lite></div>';
-    renderSchedule(globalDate, teacherObj, "teacher")
+        '<schedule-lite id="teacherschedule"></schedule-lite></div>';
+    var scheduleElement = document.getElementById('teacherschedule');
+    renderSchedule(globalDate, teacherObj, "teacher", scheduleElement)
   }
   function renderStudent(studentObj) {
     console.log(studentObj);
+    var name = studentObj.firstname + " " + studentObj.lastname;
+    var email = studentObj.firstname[0] + studentObj.lastname + "@eastsideprep.org";
```

Comment:
OK leaving this like this for now but we should really send this down from the server.